neqo-crypto/src/aead_fuzzing.rs

```diff
@@ -20,20 +20,18 @@ pub struct FuzzingAead {
 }
 
 impl FuzzingAead {
+    #[cfg(feature = "disable-encryption")]
```

Comment:
Why not disable this file completely if encryption is disabled.  That is, put this at the top:

```rust
#![cfg(feature = "disable-encryption")]
```

Then you don't have to use an `Option` on the implementation.  There is a lot that you can then remove related to the real AEAD at that point.